community.md

```diff
@@ -50,31 +50,30 @@ We run an announcement-only, no-marketing-spam mailing list that we will use to
 
 ## Want to report issues? 
 
-Oh no, you hit a bump on the road! Please tell us - we want to fix all bugs!
+Oh no, you hit a bump on the road? Please tell us - we want to fix all bugs!
 
-- Found a bug in the compiler, standard libraries or the runtime? Create an issue here: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">https://github.com/ballerina-platform/ballerina-lang/issues</a>
-- Found a bug in the language specification? Create an issue here: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">https://github.com/ballerina-platform/ballerina-spec/issues</a>
-- Found a bug on the website? Create an issue here: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">https://github.com/ballerina-platform/ballerina-dev-website/issues</a> 
+- Found a bug in the compiler, standard library, runtime, or tooling? Create an <a href="https://github.com/ballerina-platform/ballerina-lang/issues">issue</a>.

```

Comment:
Shall we swap runtime and standard library?
```suggestion
- Found a bug in the compiler, runtime, standard library, or tooling? Create an <a href="https://github.com/ballerina-platform/ballerina-lang/issues">issue</a>.
```

Reply:
IMO, we need "here" and the link should be for that.

Otherwise, we are just repeating "create an issue". We are creating an issue for all the cases, what changes is where.

Reply:
Normally, it's not a best practice in content writing to give the link saying "here". The guideline is to have a text with a hyperlink.

Reply:
+1, @pubudu91 suggested something similar.

Also, how about saying  "Create an issue in the ballerina-lang repo" instead?

Reply:
IMHO, mentioning "the repo" in the hyperlink text will be a bit redundant as the user will click the link anyway if he/she wants to create an issue irrespective of it being a GitHub repo or not.

@pubudu91 - What's your suggestion, please?

Reply:
No, I don't think Maryam meant including "repo" part in the hyperlink. 
Something like below:
"Found a bug in the compiler, standard library, runtime, or tooling? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repository."

Reply:
Yes, even saying "repository" without the hyperlink will be redundant for the same reason.  The user will click the link anyway if he/she wants to create an issue irrespective of it being a GitHub repo or not.

Reply:
ballerina-lang is just the name, reads a bit weird to me.

Reply:
+1 Changed as follows:

- Found a bug in the compiler, runtime, standard library, or tooling? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
- Found a bug in the language specification? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
- Found a bug on the website? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo.